.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1 @@
+blank_issues_enabled: false
```

Comment:
This will prevent the blank issues option from appearing.

Additionally, if we want other repos to point elsewhere for issue tracking, we could do that using this config file. https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository